crypto/rsa/rsa_backend.c

```diff
@@ -164,7 +347,7 @@ int ossl_rsa_todata(RSA *rsa, OSSL_PARAM_BLD *bld, OSSL_PARAM params[],
             || !ossl_param_build_set_multi_key_bn(bld, params,
                                                   ossl_rsa_mp_coeff_names,
                                                   coeffs))
-        goto err;
+            goto err;
```

Comment:
Is it a necessary change?

Reply:
not necessecary, but check_format.pl flagged it as a style error.